x-pack/filebeat/input/azureeventhub/v1_input.go

```diff
@@ -105,9 +109,11 @@ func (in *eventHubInputV1) Run(
 	err = in.run(ctx)
 	if err != nil {
 		in.log.Errorw("error running input", "error", err)
+		inputContext.UpdateStatus(status.Failed, err.Error())
```

Comment:
Hey @faec, I missed the opportunity to report the input status to the Elastic Agent when I updated the input to the Input API v2.

I am trying to address scenarios (like https://github.com/elastic/integrations/issues/9659) where the input starts, and after a while, it encounters a fatal error, and the SDK worker shuts down.

What are your recommendations for reporting input status to the agent? Which input should I use as a reference?

🙇